fcs/FSharp.Compiler.Service.ProjectCrackerTool/ProjectCrackerTool.fs

```diff
@@ -336,6 +337,8 @@ module internal ProjectCrackerTool =
                   if not (String.IsNullOrWhiteSpace nw) then yield "--nowarn:" + nw
               for nw in warningsAsError do
                   if not (String.IsNullOrWhiteSpace nw) then yield "--warnaserror:" + nw
+              for nw in warningsNotAsError do
+                  if not (String.IsNullOrWhiteSpace nw) then yield "--warnaswarn:" + nw
```

Comment:
This argument already exists as `--warnaserror-:`

Reply:
@saul, thanks! should I revert the changes in `src/fsharp/CompileOptions.fs` as well?